api/foxpass_whitelist_sync.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+"""
+This script requires the external libraries from requests
+pip install requests
+
+To run:
+python foxpass_whitelist_sync.py --api-key <api_key> --hostname <DDNS_hostname> --whitelist-name <whitelist_name>
```

Comment:
`whitelist` is too vague since we do group whitelisting too. `whitelist_ip`? `ip_whitelist`?

Reply:
Noted